src/services/positionService.ts

```diff
@@ -645,6 +655,16 @@ export default class PositionService {
     if (yieldFrom || yieldTo) {
       permissions = [...permissions, PERMISSIONS.TERMINATE];
     }
+    const POLYGON_WETH = '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619';
+
+    const forwardCallData = getForwardFunctionCallHelper(
```

Comment:
this would only be needed in case of using multichain right?

Reply:
Yes